run_dir/design/project_samples.html

```diff
@@ -1,584 +1,28 @@
 {% extends "base.html" %}
 
 <!--
-Template file: project_samples.html
-URL: /project/[project id]
-Title: Production Samples
-Description: Details page for a single project
+Template file: projects_samples.html
```

Comment:
Should the file name be `project_samples` or `projects_samples`?

Reply:
Fixed.